### PR TITLE
mender-connect master: depend on mender 3.2

### PR DESCRIPTION
**Pull request by lluiscampos**
Following MEN-5203, the upcoming mender-connect releases will depend on
mender 3.2 or newer.

Changelog: Title

### CODE REVIEW

**Comment by lluiscampos** [comment]
See also:
* https://github.com/mendersoftware/mender-dist-packages/pull/147

File: meta-mender-core/recipes-mender/mender-connect/mender-connect_1.2.0.bb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 require mender-connect.inc
 
+RDEPENDS_${PN} = "glib-2.0 mender-client (>= 2.5)"
```
Comment (kacf) [inline meta-mender-core/recipes-mender/mender-connect/mender-connect_1.2.0.bb:3]: Very often, we just copy the old file. How is the best way to remember to remove this? A task set for Mender 3.2?

**Review by kacf — APPROVED** [review]

File: meta-mender-core/recipes-mender/mender-connect/mender-connect_1.2.0.bb (hunk 1 above)
Comment (lluiscampos) [reply meta-mender-core/recipes-mender/mender-connect/mender-connect_1.2.0.bb:3]: Yes, I thought about that too. I am creating a reminder task, good idea!

Bu the way, I also considered to leave the `inc` with the dependency on `mender-client (>= 2.5)` and update only the master recipe... but decided for this approach which is more verbose but cleaner in my opinion.

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline :ship:

File: meta-mender-core/recipes-mender/mender-connect/mender-connect_1.2.0.bb (hunk 1 above)
Comment (lluiscampos) [reply meta-mender-core/recipes-mender/mender-connect/mender-connect_1.2.0.bb:3]: Created [MEN-5229](https://tracker.mender.io/browse/MEN-5229) :+1:

**Pull request merged**